packages/site/src/editorBundle.js

```diff
@@ -1,9 +1,11 @@
 import React, { useEffect, useRef, useState } from "react";
 import ReactDOM from "react-dom";
 import { useFormState } from "@jobber/hooks";
+import { updateTheme } from "@jobber/components/AtlantisThemeContext";
 
 window.React = React;
 window.ReactDOM = ReactDOM;
+window.updateTheme = updateTheme;
```

Comment:
Without this, the component preview wouldn't work (error in console: `ReferenceError: updateTheme is not defined`) and pointed to `editorBundle`

![Kapture 2025-01-30 at 22 49 23](https://github.com/user-attachments/assets/b25d3926-8eef-419e-9511-b4c71447b2e0)